File: JZResgate.Front/src/app/models/recovery-truck.ts
export class RecoveryTruck {
  alias: string;
  model: string;
  plate: string;
  id: string;
  createDate: Date;

  constructor(alias, model, plate, id = null, createdDate = null) {
    this.alias = alias;
    this.model = model;
    this.plate = plate;
    this.id = id;
    this.createDate = createdDate;
  }
}

File: JZResgate.Api/JZResgate.Domain/Models/RecoveryTruck.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace JZResgate.Domain.Models
{
    public class RecoveryTruck: BaseModel
    {
        public string Alias { get; set; }
        public string Plate { get; set; }
        public string Model { get; set; }
        public DateTime CreateDate { get; set; }
    }
}
